dll/shellext/mydocs/lang/ro-RO.rc

```diff
@@ -0,0 +1,13 @@
+/*
+ * PROJECT:     ReactOS My Documents Shell Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2022 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
+
+LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+{
+    IDS_NOSRCFILEFOUND "Fișierul sau dosarul sursă '%ls' nu a fost găsit."
+}
```

Comment:
`ro-RO.rc` file is created, but not added to the place where other .rc files are present.